Extension/ASPNETCore-EssentialJS2-Extension/Visual-Studio/Syncfusion-Project-Templates.md

```diff
@@ -27,7 +27,7 @@ The following steps is used to create the **Syncfusion ASP.NET Core (Essential J
 
     > From Visual Studio 2019, Syncfusion menu is available under Extension in Visual Studio menu.
 
-    ![new project](images/SyncfusionMenu.png)
+    ![syncfusion menu in new project for Extension ASP.NET Core essentialjs2 project template](images/aspnet-core-essentialjs2-extension-with-project-templates-syncfusion-menu.png)

```

Comment:
Alt text character count is too high. Change the text like as below,
Create new project using ASP.NET Core template.
Use the same content to image name.
create-new-project-using-aspnet-core-template.

Reply:
Resolved